src/freertos_drivers/st/Stm32Can.cxx

```diff
@@ -36,6 +36,7 @@
 #include "Stm32Can.hxx"
 
 #include <stdint.h>
+#include <string.h>
```

Comment:
This include might be able to be removed if the programatic select of device instance by name is also removed.

Reply:
Done. (removed unused include)